bundles/org.openhab.binding.jablotron/src/main/resources/ESH-INF/config/config.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<config-description:config-descriptions
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:config-description="https://openhab.org/schemas/config-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/config-description/v1.0.0 https://openhab.org/schemas/config-description-1.0.0.xsd">
+
+	<config-description uri="thing-type:jablotron:device">
+		<parameter name="refresh" type="integer" required="false">
```

Comment:
Better specify a reasonable min value (e.g. 1) to prevent entering 0 or a negative number, which would crash your binding. Same for below.
```suggestion
		<parameter name="refresh" type="integer" required="false" unit="s">
```

Reply:
added unit as well as min value